src/App.tsx

```diff
@@ -1,61 +1,40 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import { NewTodoForm } from './components/newTodoForm/newTodoForm';
+import { Todo } from './types';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const getPreperedTodos = (todos: Todo[]) => {
+  return todos.map(todo => {
+    const user = usersFromServer.find(u => u.id === todo.userId);
+
+    return { ...todo, user };
+  });
+};
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const preparedTodos: Todo[] = getPreperedTodos(todos);
```

Comment:
Make sure to update the function call to `getPreparedTodos` after correcting the typo in the function name.